src/dbspro/rules.smk

```diff
@@ -5,13 +5,15 @@ import os
 from dbspro.utils import get_abcs
 
 # Read sample and handles files.
-abc = get_abcs(config["abc_sequences"])
-handles = pd.read_csv(config["handles"], sep='\t').set_index("Name", drop=False)
+abc = get_abcs("ABC-sequences.fasta")
+print(abc)
```

Comment:
Did you forget to remove this print statment after debugging the script or is there a use to having it here.

Reply:
Woops!

Reply:
Hm, again - here my local installation is out of sync I see now. Strange, but either way I'll patch it.